src/encoded/schemas/analyte.json

```diff
@@ -23,6 +23,26 @@
         "rna_integrity_number_instrument": [
             "rna_integrity_number"
         ],
+        "dna_integrity_number": [
+            "dna_integrity_number_instrument"
+        ],
+        "dna_integrity_number_instrument": [
+            "dna_integrity_number"
+        ],
+        "genomic_quality_number": [
+            "genomic_quality_number_instrument",
+            "genomic_quality_size_threshold"
+        ],
+        "genomic_quality_number_instrument": [
+            "genomic_quality_number"
+        ],
+        "dna_quality_number": [
+            "dna_quality_number_instrument",
+            "dna_quality_size_threshold"
+        ],
+        "dna_quality_number_instrument": [
+            "dna_quality_number"
+        ],
```

Comment:
Strongly consider adding tests for these.

Reply:
Added tests for dna_integerity, genomic_quality, and dna_quality. I also make all three combinations mutually required. Thanks!